src/common/actions/register-name.js

```diff
@@ -254,14 +254,12 @@ export async function internalNameOwnership(root, discharge, snapName) {
   }
 }
 
-export function checkNameOwnership(repository, snapName) {
-  if (!repository || !snapName) {
-    throw new Error('`repository` and `snapName` are required params of `checkNameOwnership`');
+export function checkNameOwnership(id, snapName) {
+  if (!id || !snapName) {
```

Comment:
We were passing a `repo` as a param only to take `url` of it and use as snap id, so it simplifies it.